src/react-components/src/components/01-atoms/Button/Button.tsx

```diff
@@ -8,14 +8,18 @@ export interface ButtonProps {
   id: string;
   /** The action to perform on the <button>'s onClick function */
   callback: (event: React.MouseEvent) => void;
-  content: string | JSX.Element;
+  /** The content to render inside the the button. An alternative
+   * to passing children elements. */
+  content?: string | JSX.Element;
```

Comment:
do you think this is necessary?  Can we just say you gotta pass it as children?

Reply:
I don't think so, but I didn't know if you _still_ wanted to support `content`, so I left it there.

Reply:
technically this is a breaking change but it's not used in RN so it can be removed.  We can make a new ticket for it though, no need to expand this one.

Reply:
Sure. I'll leave it in here for now then.